### ISSUE TITLE
Formula One 2001 (Europe) (Es,It) SCES-03424 crash

### ISSUE THREAD

**Issue by Nariturbina** [stenzek/duckstation#2643]
The car accelerates too much and when colliding it bounces anywhere. The car has an erratic behavior that makes it unplayable. Sorry to have returned the topic, but it had been closed and the error persists. Your emulator is excellent, greetings!

**Comment by liljotik** [comment]
The game has force interpreter on in per game settings to prevent that. You must be launching the game from the desktop instead of a game directory which makes per game settings not work. Try launching the game from the list and see if that fixes it.

![interpe](https://user-images.githubusercontent.com/92196998/136643727-aa2d4429-d34a-4579-8c9b-09077ff2d1d9.PNG)

![la](https://user-images.githubusercontent.com/92196998/136643757-836a0587-afe3-4013-8fb3-56a23ca8ca23.PNG)

**Comment by Nariturbina** [comment]
Hello, thanks for answering! If it worked, as I don't have "Apply Per-Game Setting" activated, I went to Console Settings and left it as the screenshots show:
![Captura1](https://user-images.githubusercontent.com/92188562/136730575-2eb98bc6-371f-4537-b679-0f532551b219.PNG)
![Captura2](https://user-images.githubusercontent.com/92188562/136730580-0c46e8ae-0659-4c63-8767-41c5304faef2.PNG)

**Comment by Nariturbina** [comment]
No, because if I configure it like this, the points that mark the positions of the other pilots on the map appear in a row:
![Clip](https://user-images.githubusercontent.com/92188562/136791946-89049b89-a11f-410f-a3bf-87f6810d4e1c.jpg)

**Comment by liljotik** [comment]
I don't know what on earth you're doing to make that happen but it's not caused by per game settings. Does look like you're using a widscreen hack though. Try turning that off. 

![SCES-03524_2021-10-12_01-09-36](https://user-images.githubusercontent.com/92196998/136805004-436ae737-5c98-4375-9648-d2c5d7a94b67.png)

**Comment by Nariturbina** [comment]
Ready, I have corrected the configuration and the game is fine, thank you very much!!
